vips/foreign.c

```diff
@@ -1,29 +1,92 @@
 #include "lang.h"
 #include "foreign.h"
+#include "govips.h"
 
-int load_image_buffer(void *buf, size_t len, int imageType, VipsImage **out) {
+int load_image_buffer_operation(VipsOperation *operation, VipsBlob *blob, VipsImage **out, ImportParams *params) {
+    int result;
+
+    // Set operation parameters
+    result = vips_object_set(VIPS_OBJECT(operation), "buffer", blob, "out", out, NULL);
+    if (result) {
+        return result;
```

Comment:
why no log for this?

Reply:
The corresponding message will be inside the error.